script.js

```diff
@@ -0,0 +1,32 @@
+let basic_monthly = document.getElementById("basic_monthly");
+let basic_annualy = document.getElementById("basic_annualy");
+
+let professioanal_monthly = document.getElementById("professioanal_monthly");
+let professioanal_annualy = document.getElementById("professioanal_annualy");
+
+let master_monthly = document.getElementById("master_monthly");
+let master_annualy = document.getElementById("master_annualy");
+
+let toogle = document.getElementById("toogle_button");
+
+basic_monthly.style.display = "none";
+professioanal_monthly.style.display = "none";
+master_monthly.style.display = "none";
+
+toogle.addEventListener("click", (e) => {
+  if (toogle.checked === true) {
+    basic_monthly.style.display = "";
+    professioanal_monthly.style.display = "";
+    master_monthly.style.display = "";
```

Comment:
Instead of setting the display value to `""` you can set it to a correct value that exists in CSS 
read https://developer.mozilla.org/en-US/docs/Web/CSS/display